doc/source/notebooks/advanced/multioutput.ipynb

```diff
@@ -0,0 +1,747 @@
+   "source": [
+    "This notebook will show how to construct a multi-output GP model using GPflow. We will consider a regression problem for functions $f: \\mathbb{R}^D \\rightarrow \\mathbb{R}^P$. We assume that the dataset is of the form $(X, f_1), \\dots, (X, f_P)$, i.e. we observe all the outputs for a given input location. For the case where we do fully observe all the outputs for a given input please see the [coregionalisation notebook](./coregionalisation.ipynb).\n",
+    "\n",
+    "Here we assume a model of the form: \n",
+    "$$f(x) = W g(x),$$\n",
+    "where $g(x) \\in \\mathbb{R}^L$, $f(x) \\in \\mathbb{R}^P$ and $W \\in \\mathbb{R}^{P \\times L}$. We assume that the outputs of $g$ are uncorrelated, and by *mixing* them with $W$ they become correlated. In this notebook, we illustrate how to build this model using SVGP for $g$, which scales well with the numbers of datapoints and outputs. \n",
+    "\n",
+    "Here we have two cases for $g$:\n",
+    "1. The output dimensions of $g$ share the same kernel.\n",
+    "2. Each output of $g$ has a separate kernel.\n",
+    "\n",
+    "\n",
+    "In addition, we have two further subcases for the inducing inputs of $g$:\n",
+    "1. The $g$'s share the same inducing inputs.\n",
+    "2. Each output of $g$ has its own set of inducing inputs.\n",
+    "\n",
+    "Notation:\n",
+    "$\n",
+    "\\newcommand{\\GP}{\\mathcal{GP}}\n",
+    "\\newcommand{\\NN}{\\mathcal{N}}\n",
+    "\\newcommand{\\LL}{\\mathcal{L}}\n",
+    "\\newcommand{\\RR}{\\mathbb{R}}\n",
+    "\\newcommand{\\EE}{\\mathbb{E}}\n",
+    "\\newcommand{\\valpha}{\\boldsymbol\\alpha}\n",
+    "\\newcommand{\\vf}{\\mathbf{f}}\n",
+    "\\newcommand{\\vF}{\\mathbf{F}}\n",
+    "\\newcommand{\\vg}{\\mathbf{g}}\n",
+    "\\newcommand{\\vW}{\\mathbf{W}}\n",
+    "\\newcommand{\\vI}{\\mathbf{I}}\n",
+    "\\newcommand{\\vZ}{\\mathbf{Z}}\n",
+    "\\newcommand{\\vu}{\\mathbf{u}}\n",
+    "\\newcommand{\\vU}{\\mathbf{U}}\n",
+    "\\newcommand{\\vX}{\\mathbf{X}}\n",
+    "\\newcommand{\\vY}{\\mathbf{Y}}\n",
+    "\\newcommand{\\identity}{\\mathbb{I}}\n",
+    "$\n",
+    "- $X \\in \\mathbb{R}^{N \\times D}$ the input\n",
+    "- $Y \\in \\RR^{N \\times P}$ the output\n",
+    "- $k_{1..L}$, $L$ kernels on $\\RR^{N \\times D}$\n",
+    "- $g_{1..L}$, $L$ independent $\\GP$s  with $g_l \\sim \\GP(0,k_l)$\n",
+    "- $f_{1..P}$, $P$ correlated  $\\GP$s  with $\\vf = \\vW \\vg$ "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2018-06-19T10:58:45.195606Z",
+     "start_time": "2018-06-19T10:58:44.172232Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import gpflow as gpf\n",
+    "\n",
+    "import gpflow.multioutput.kernels as mk\n",
+    "import gpflow.multioutput.features as mf\n",
+    "%matplotlib inline\n",
+    "\n",
+    "MAXITER = gpf.test_util.notebook_niter(int(15e1))"
```

Comment:
```suggestion
    "MAXITER = gpf.test_util.notebook_niter(150)"
```
or do you mean something other than 15e1? 15e3? even then, it's still easier to parse to write out 15000 :)